src/worker.cpp

```diff
@@ -742,6 +742,9 @@ bool Worker::primeInventory(const std::string& i_vpdFilePath)
             continue;
         }
 
+        vpdSpecificUtility::resetDataOnDbus(std::string(l_Fru["inventoryPath"]),
```

Comment:
Add a comment stating that clearing data from Dbus if already exists.

Reply:
Done